lib/trento/abilities/ability.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Trento.Abilities.Ability do
+  @moduledoc """
```

Comment:
Let's add a docstring or put `@moduledoc false`.
I would maybe document that the `label` is just an informative field to describe what the ability does